target/linux/ramips/dts/mt7620a_netgear_ex6120.dts

```diff
@@ -0,0 +1,58 @@
+/* This file is released into the public domain */
+
+/dts-v1/;
+
+#include "mt7620a_netgear_ex3700_ex6130.dtsi"
+
+/ {
+	compatible = "netgear,ex6120", "ralink,mt7620a-soc";
+	model = "Netgear EX6120";
+
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_green;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_green;
+		label-mac-device = &ethernet;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_green: power_g {
+			label = "ex6120:green:power";
+			gpios = <&gpio2 23 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		power_a {
```

Comment:
power_amber
Same below ...

Reply:
Not present in target/linux/ramips/dts/mt7620a_netgear_ex6130.dts. If you want this changed, please change target/linux/ramips/dts/mt7620a_netgear_ex6130.dts first. I'm not fiddling with code that I works, is identical to existing code, and might break things I do not even know exist.